src/user/user.helpers.ts

```diff
@@ -0,0 +1,95 @@
+import HttpException from '../exceptions/HttpException';
+import UserNotFoundException from '../exceptions/UserNotFoundException';
+import { IHelperArgs } from '../interfaces/helpers.interface';
+import { IUser } from './user.interface';
+import { v4 as uuid_v4 } from 'uuid';
+import * as utils from '../utils';
+import InvalidIDException from '../exceptions/InvalidIDException';
+
+export const getUserByIdHelper = async (userHelpersArgs: IHelperArgs<IUser>): Promise<void> => {
+    const { db, request, response, next } = userHelpersArgs;
+    const { id } = request.params;
+    try {
+        const user = await db.getById(id);
+        if (user) {
```

Comment:
To avoid if-else statement nesting, consider using "early-return" approach.
E.g. handle the UserNotFoundException before defining any other logic.